src/settings.js

```diff
@@ -1232,10 +1232,17 @@ var IN_TEST_HARNESS = 0;
 // If true, enables support for pthreads.
 var USE_PTHREADS = 0;
 
-// Specifies the number of web workers that are preallocated before runtime is
-// initialized. If 0, workers are created on demand.
+// Specifies the number of web workers that are preallocated with the
```

Comment:
Maybe best to avoid the word "preallocated" here, as we use it for the other group now in the JS.

Reply:
Done. Using the word created.